data/profiles/solarstorm2012.profile

```diff
@@ -0,0 +1,124 @@
+{
+  "actions": [
+    {
+      "documentation": "Reset button. Sets time to start at start of dataset. No loop",
+      "gui_path": "2012July",
+      "identifier": "2012July.Reset-loop",
+      "is_local": false,
+      "name": "Reset button.",
+      "script": "openspace.time.setTime('2012-JUL-01 07:00:00.00');\nopenspace.scriptScheduler.clear();"
+    },
+    {
+      "documentation": "Makes the sun turn black",
+      "gui_path": "2012July",
+      "identifier": "2012July.Dark-sun",
+      "is_local": false,
+      "name": "Dark sun",
+      "script": "openspace.setPropertyValueSingle('Scene.Sun.Renderable.Layers.ColorLayers.Texture.Settings.Multiplier', 0.000000);"
+    },
+    {
+      "documentation": "Sets time to start of data. sets high delta time. loops from start when at end of data",
```

Comment:
-> "Sets time to start of data, sets high delta time, and loops from start when at end of data"